doc/man1/openssl-verification-options.pod

```diff
@@ -18,31 +18,56 @@ L<X509_verify_cert(3)> verify X.509 certificates. The details of how each
 command handles errors are documented on the specific command page.
 
 Certificate verification is a complicated process, consisting of
-a number of separate steps that are detailed in the following paragraphs.
+a number of separate steps that are detailed in the following sections.
+
+=head2 Certification Path Building
 
 First, a certificate chain is built up starting from the target certificate
 and typically ending in a self-signed "root" CA certificate.
 It is an error if the whole chain cannot be built up
 unless the B<-partial_chain> option is given.
+
 The chain is built up iteratively, looking up in turn
-the certificate of the signer ("issuer") of the current certificate.
+a currently valid (non-expired) certificate with suitable key usage that
+matches as an issuer of the current "subject" certificate as described below.
+In each iteration the first certificate found this way is taken.
+For efficiency, no backtracking is performed, thus
+any further candidate issuer certificates that would match equally are ignored.
```

Comment:
Is this true?  I thought we had alternate path construction for some time (since da084a5ec6cebd67ae27f2463ebe4a50bb840fa5?  Though I thought it was a more recent addition than 2015...)

Reply:
> I thought we had alternate path construction for some time (since da084a5ec6cebd67ae27f2463ebe4a50bb840fa5)? 
 
Interesting to learn more and more about the history of the cert patch construction implementation.
Yes, that commit da084a5ec6cebd67ae27f2463ebe4a50bb840fa5 introduced some flexibility, but soon thereafter the `X509_V_FLAG_NO_ALT_CHAINS` option was introduced, and a year later, 
commit 0daccd4dc1f1ac62181738a91714f35472e50f3c introduced `X509_V_FLAG_TRUSTED_FIRST` for version 1.1.0 in a way that apparently cannot be disabled (at least if we can trust its documentation), and this disabled the feature again (see assignments to `may_alternate` in `build_chain()`.

BTW, my new PR #13748 offers a more general and elegant solution to the problem addressed by commit da084a5ec6cebd67ae27f2463ebe4a50bb840fa5: 
simply stop chain building successfully as soon as you hit a suitable trust anchor.

Reply:
That new PR *changes existing behavior.*  Please do not put this into 3.0 now.

Reply:
@richsalz, your comment "That new PR changes existing behavior. Please do not put this into 3.0 now." 
belongs to #13748, not here. It is being addressed there.